### PR TITLE
Happy Thoughts project by Katsiez

### PR DESCRIPTION
**Pull request by Katsiez**
Link to my deployed project: https://happy-thoughts-kat.netlify.app/

My reflections on how this week's project turned out: I was so worried, but I LOVED this week's project! It was so much fun to build. 😭 🤍

Things I'd like to have clarified or explained in more detail: Nothing much other than the fact that my heart button appears to be not centered on my phone, kind of weird. Nothing major! I feel super happy about how far I was able to come and have even started working on black level requirements. SO FUN!

### CODE REVIEW

**Review by jonnaru — APPROVED** [review]
Really cool, Kat!!! 👏 It is interesting to read though your code and see your solutions:) You should really go back to YourLikes.js component when you have time. I was super curious to see it! Your code is clean and easy to read 🤩 The only thing I left comments on was the lines with several spans. However, great work! Looks amazing!

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,50 @@
...
+          .then(json => setThoughts(json))
+      })
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <main>
+      <h1><span className="heart" role="img" aria-label="purple-heart">{"🤍"}</span>HAPPY THOUGHTS<span className="heart" role="img" aria-label="purple-heart">{"🤍"}</span></h1>
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply code/src/App.js:40]: Yep :)

File: code/src/components/HappyForm.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            headers: { "Content-Type": "application/json" }
+        })
+            .then(() => {
+                setMessage("")
+                props.onFormSubmit(message)
+            })
+            .catch(err => console.log("error:", err))
+    }
+
+    const emptyMessage = (value) => value.replace(/\s/g, "").length === 0;
```
Comment (EmOlsz) [inline code/src/components/HappyForm.js:25]: `isMessageEmpty` would be better from the semantic point of view of how we name boolean functions

File: code/src/components/HappyThoughts.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import moment from "moment"
+
+import "./happyThoughts.css"
+
+export const HappyThoughts = ({ thought, onLiked }) => {
+    console.log(thought, onLiked)
```
Comment (EmOlsz) [inline code/src/components/HappyThoughts.js:7]: Delete all console.logs before final push

**Pull request closed without merging**